examples/cms/src/FieldMapping.tsx

```diff
@@ -0,0 +1,235 @@
+import type { ManagedCollection, ManagedCollectionField } from "framer-plugin"
+import type { DataSource, FieldIds, ManagedCollectionFields } from "./data"
+
+import { framer } from "framer-plugin"
+import { useState, useMemo, useEffect, memo } from "react"
+import { computeFieldsFromDataSource, mergeFieldsWithExistingFields, syncCollection } from "./data"
+import { Spinner } from "./components/Spinner"
+
+function ChevronIcon() {
+    return (
+        <svg xmlns="http://www.w3.org/2000/svg" width="8" height="16">
+            <path
+                d="M 3 11 L 6 8 L 3 5"
+                fill="transparent"
+                strokeWidth="1.5"
+                stroke="#999"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+            />
+        </svg>
+    )
+}
+
+interface FieldMappingRowProps {
+    field: ManagedCollectionField
+    disabled: boolean
+    onDisable: (fieldId: string) => void
```

Comment:
`onDisable`, sounds like it will only be used to disable the field. I would name it `onToggleDisabled`